addons/crm/models/crm_team.py

```diff
@@ -170,11 +208,25 @@ def _graph_get_dates(self, today):
             return [start_date, end_date]
         return super(Team, self)._graph_get_dates(today)
 
+    def _get_graph(self):
+        graph_datas = super(Team, self)._get_graph()
+        if self.dashboard_graph_model == 'pipeline' and self.dashboard_graph_group_pipeline == 'stage':
+            CrmStage = self.env['crm.stage']
+            graph_datas[0]['values'] = sorted(graph_datas[0]['values'], key=lambda el: CrmStage.browse(el['label']).sequence)
```

Comment:
Stage sequence and name isn't available in graph_data as it's not a generic case that's handled in the sales_team module.
Do you mean to extract the browsing to improve performance ? That could be done I guess.